### ISSUE TITLE
Enh Shaman changing priority list during opener

### ISSUE THREAD

**Issue by jakeparshall** [Hekili/hekili#4123]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

Shaman - Enhancement

### Describe the Issue

https://www.dropbox.com/s/vtok7r4l94wh4uq/Screen%20Recording%202024-10-29%20at%208.16.06%20PM.mov?dl=0

I Pwave to get the pull started and when i get into mele the priority changes to go straight into ascendance, Skipping the Simc Prio of Lava lash, Doom winds, Cash lightening etc. 

Just re tested and i always does this regardless of distance from the target when starting pull. I can stand out of range and the correct actions will populate, but when i get into range of the enemies it changes priority. 

### How to Reproduce

1. Enter game as Enh shaman
2. Stand out of combat and look at priority list populated by Hekili
3. Fire a Pwave at 5 target dummies 
4. Walk towards Mele and priority list changes

It seems to be whenever you enter in range of your detected targets

### Snapshot (Link)

https://pastebin.com/NdMfe8Xk

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/gUwVrcwGbrPkL8ib5pWnvh

### Additional Information

TLDR: Walking into range of enemies changes the prio list

**Comment by Hekili** [comment]
> It seems to be whenever you enter in range of your detected targets

That's when those targets begin to count for target detection, and your Thorim's Invocation is already primed with Chain Lightning, so you're not starting from the same point that the sim does (TI starts out set to Lightning Bolt).
